config.go

```diff
@@ -3,10 +3,11 @@ package ffclient
 import (
 	"context"
 	"errors"
-	"github.com/thomaspoignant/go-feature-flag/ffnotifier"
 	"log"
 	"time"
 
+	"github.com/thomaspoignant/go-feature-flag/ffnotifier"
```

Comment:
Can we keep the import as before, there is no change in them just a different order.